### PR TITLE
Rename ensa modules, remove connector form ERS

### PR DESCRIPTION
**Pull request by lilyeyes**
Rename ensa modules.
Remove connector form ERS node.
Check lsn command output to confirm it lists the nodes configured in the test.

- Related ticket:
[TEAM-9629](https://jira.suse.com/browse/TEAM-9629) - [on prem][QAM] Evaluate *sles4sap/sap_suse_cluster_connector* test module in ENSA2 Scenario
[TEAM-9633](https://jira.suse.com/browse/TEAM-9633) - Rename ensa modules "netweaver_ensa2_cluster_connector" and "sap_suse_cluster_connector"
- Verification run (search `Found` for the `lsn` output checking results): 
15-SP7 sles4sap_ensa_ascs_node: https://openqa.suse.de/tests/15425974#dependencies (passed)
15-SP6 sles4sap_ensa_ascs_node: http://openqa.suse.de/tests/15426165#dependencies (passed)
12-sp5 sles4sap_ensa_ascs_node: http://openqa.suse.de/tests/15425978#dependencies (passed)
15-SP7 sles4sap_nw_node01: http://openqa.suse.de/tests/15425984#dependencies (passed)

### CODE REVIEW

File: tests/sles4sap/sap_suse_cluster_connector.pm [hunk 1]
```diff
@@ -77,11 +77,21 @@ sub run {
...
+        validate_script_output("cat $log_file | cut -d : -f 4", sub { m/$remote_node/ });
+        record_info("Found $remote_node in lsn output");
     }
 
     # Test Stop/Start of SAP resource
-    my $hana_resource_name = get_var('USE_SAP_HANA_SR_ANGI') ? "rsc_SAPHanaCtl_${instance_sid}_$instance_type$instance_id" : "rsc_SAPHana_${instance_sid}_$instance_type$instance_id";
+    my $hana_resource_name
+      = get_var('USE_SAP_HANA_SR_ANGI')
+      ? "rsc_SAPHanaCtl_${instance_sid}_$instance_type$instance_id"
+      : "rsc_SAPHana_${instance_sid}_$instance_type$instance_id";
```
Comment (lilyeyes) [inline tests/sles4sap/sap_suse_cluster_connector.pm:94]: These 4 lines are for fixing `tidy` error

File: tests/sles4sap/sap_suse_cluster_connector.pm [hunk 2]
```diff
@@ -77,11 +77,21 @@ sub run {
         my $rsc = "rsc_${rsc_type}_${instance_sid}_$instance_type$instance_id";
         wait_for_idle_cluster;
         exec_conn_cmd(binary => $binary, cmd => "lsn --res $rsc", log_file => $log_file);
-        validate_script_output("cat $log_file 2>&1", sub { m/$rsc/ });
+        # Check the "node list" contains localhost
+        my $hostname = get_required_var('HOSTNAME');
+        validate_script_output("cat $log_file | cut -d : -f 4", sub { m/$hostname/ });
+        record_info("Found $hostname in lsn output");
+        # Check the "node list" contains remote node
+        my $remote_node = script_output("crm status bynode | grep 'Node.*online' | cut -d : -f 1 | grep -v $hostname | rev | cut -d ' ' -f 1 | rev");
```
Comment (badboywj) [inline tests/sles4sap/sap_suse_cluster_connector.pm:85]: this is short `crm status bynode | grep -Po '(?<=\* Node )(.*)(?=: online)' | grep -v $hostname`, and it is just suggestion and need verification, you can have a try if you want

**Review by Amrysliu — APPROVED** [review]
LGTM

File: tests/sles4sap/sap_suse_cluster_connector.pm (hunk 2 above)
Comment (lilyeyes) [reply tests/sles4sap/sap_suse_cluster_connector.pm:85]: Tried  but failed, also I think your way is shorter but hard to read/understand so I would like to keep my way as it is simple:
```
> cat a
  * Node ensa-node01: online:
dddd
  * Node ensa-node02: online:
bbb
llzhao@localhost:/tmp> cat a  | grep -Po '(?<=\* Node )(.*)(?=: online$)' | grep -v node01
echo $?
1

```

Comment (badboywj) [reply tests/sles4sap/sap_suse_cluster_connector.pm:85]: `crm status bynode | grep -Po '(?<=\* Node )(.*)(?=: online:$)' | grep -v $hostname` is correct, I lost ":". and it is OK, regex is difficult to understand

**Review by mpagot — APPROVED** [review]
LGTM

File: tests/sles4sap/sap_suse_cluster_connector.pm (hunk 2 above)
Comment (lilyeyes) [reply tests/sles4sap/sap_suse_cluster_connector.pm]: Updated code (`"crm status bynode | grep -Po '(?<=\\* Node )(.*)(?=: online:\$)' | grep -v $hostname"`) and VR is good: https://openqa.suse.de/tests/15461873#step/sap_suse_cluster_connector/72

**Review by badboywj — APPROVED** [review]
LGTM, thanks

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Pull request merged**